netctl/netctl.go

```diff
@@ -252,6 +252,71 @@ func listNetworks(ctx *cli.Context) {
 	}
 }
 
+func createTenant(ctx *cli.Context) {
+	argCheck(1, ctx)
+
+	tenant := ctx.Args()[0]
+	subnetPool := ctx.String("subnet-pool")
+	subnetLen := ctx.Int("subnet-len")
+	vlans := ctx.String("vlans")
+	vxlans := ctx.String("vxlans")
+
+	if subnetPool == "" || vlans == "" || vxlans == "" {
+		errExit(ctx, exitHelp, "Invalid Arguments", true)
+	}
+
+	logrus.Infof("Creating tenant: %s", tenant)
+
+	url := fmt.Sprintf("%s%s/", tenantURL(ctx), tenant)
+	args := map[string]interface{}{
+		"key":        tenant,
+		"tenantName": tenant,
+		"subnetPool": subnetPool,
+		"subnetLen":  subnetLen,
+		"vlans":      vlans,
+		"vxlans":     vxlans,
+	}
+
+	postMap(ctx, url, args)
+}
+
+func deleteTenant(ctx *cli.Context) {
+	argCheck(1, ctx)
+
+	tenant := ctx.Args()[0]
+
+	logrus.Infof("Deleting tenant %s", tenant)
+
+	url := fmt.Sprintf("%s%s/", tenantURL(ctx), tenant)
+	deleteURL(ctx, url)
+}
+
+func listTenants(ctx *cli.Context) {
+	argCheck(0, ctx)
+
+	list := getList(ctx, tenantURL(ctx))
+
+	if ctx.Bool("json") {
+		dumpList(ctx, list)
```

Comment:
minor, if we add a `return` in the `if` block, then we can get rid of the `else` block and so the indentation.